getting-started-with-api/java-sdk/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>smartling-samples</groupId>
+  <artifactId>getting-started</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>getting-started</name>
+  <url>https://help.smartling.com/hc/en-us/articles/1260804661570-Getting-Started</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
```

Comment:
Set something modern. Probably it will not add any benefits for the current (very simple) code, but it will look like - this is up to date code.

Reply:
I agree, let's set 11 version of Java because even 1.8 is outdated now

Reply:
Okay, will change to 11.